Kiosk/Bid Fulfillment/RegistrationPasswordViewController.swift

```diff
@@ -4,12 +4,16 @@ import Moya
 import Action
 
 class RegistrationPasswordViewController: UIViewController, RegistrationSubController {
+    
     @IBOutlet var passwordTextField: TextField!
     @IBOutlet var confirmButton: ActionButton!
     @IBOutlet var subtitleLabel: UILabel!
     @IBOutlet var forgotPasswordButton: UIButton!
+
     let finished = PublishSubject<Void>()
 
+    var provider: Networking!
```

Comment:
I think _every_ use of `var` fits under that umbrella :smile_cat: Unless something is marked as `dynamic`.

You're totally right about why its used – not sure what the best approach for documenting it would be though.